diesel.toml

```diff
@@ -0,0 +1,9 @@
+# For documentation on how to configure this file,
+# see https://diesel.rs/guides/configuring-diesel-cli
+
+[print_schema]
+file = "fastn-core/src/schema.rs"
+custom_type_derives = ["diesel::query_builder::QueryId"]
+
+[migrations_directory]
+dir = "migrations"
```

Comment:
Cool! I think they can take care of migration stuff for us.

Reply:
yeah, I was thinking of wrapping their migration feature for `fastn migrate` command :smiling_imp:

Reply:
This will be awesome.

Reply:
This will be addressed in a separate PR